RenderStatePackagerLib/interface/SerializationDeviceDescriptorParser.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
There seems to be no need for this interface. It is specific for RenderStatePackager and is not used anywhere else.